src/bin/cargo/commands/fetch.rs

```diff
@@ -9,6 +9,7 @@ pub fn cli() -> Command {
         .arg_silent_suggestion()
         .arg_target_triple("Fetch dependencies for the target triple")
         .arg_manifest_path()
+        .arg(flag("offline", "Run without accessing the network").hide(true))
```

Comment:
This *could* be used to test if all packages are fetched. Unsure if its worth showing for that reason or not.

If we hide it, should we also have the command say its deprecated when used?

Reply:
Although my first impression is that this option is really not used, I think I'm solving the wrong problem.